app/templates/gulpfile.js

```diff
@@ -139,8 +143,14 @@ gulp.task('browser-sync', ['html', 'styles', 'scripts'], function () {
   });
 });
 
+gulp.task('copy-styleguide', function() {
+  gulp.src(paths.styleguide)
+  	.pipe(replace(/---(.|\n)*---/, ''))
+  	.pipe(gulp.dest('app/_includes/components'));
+});
+
 gulp.task('serve', function () {
-  runSequence('clean', ['browser-sync', 'watch']);
+  runSequence('copy-styleguide', 'clean', ['browser-sync', 'watch']);
```

Comment:
You'll likely want to run the `copy-styleguide` task within the `default` Gulp task, as that is where the source is actually compiled.